descriptions/platform.json

```
@@ -2,28 +2,35 @@
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "type": "object",
     "title": "Platform description",
-    "description": "A description of a platform including metadata of the platform itself, available sensors and produced observations",
+    "description": "A description of a platform including metadata of the platform itself and hosted devices",
     "required": [
         "platform_id",
-        "sensors"
+        "devices"
     ],
     "properties": {
         "platform_id": {
             "type": "string",
             "title": "Platform id",
-            "desciption": "Id of the platform"
+            "description": "Id of the platform"
         },
         "metadata": {
             "type": "object",
             "title": "Metadata",
             "description": "Metadata of this platform"
         },
```

In case we want to categorize the platforms 

```suggestion
        },
      "platform_type": {
          "type": "string",
          "title": "Platform type",
          "enum": ["vessel", "land-sensor-station", "remote-control-center"],
          "description": "Cateforization of the platform"
    },
```

---

Doesn't sound unreasonable to have types for the platforms.